packages/state-manager/src/sagas/files/files.selectors.ts

```diff
@@ -4,35 +4,13 @@ import { currentChannelMessages } from '../publicChannels/publicChannels.selecto
 import { StoreKeys } from '../store.keys'
 import { CreatedSelectors, StoreState } from '../store.types'
 import { downloadStatusAdapter } from './files.adapter'
-import { DownloadState, DownloadStatus } from './files.types'
 
 const filesSlice: CreatedSelectors[StoreKeys.Files] = (state: StoreState) => state[StoreKeys.Files]
 
-export const selectDownloadStatuses = createSelector(filesSlice, state =>
+export const downloadStatuses = createSelector(filesSlice, state =>
   downloadStatusAdapter.getSelectors().selectEntities(state.downloadStatus)
 )
 
-// Map download statuses to currently displayed messages
-export const downloadStatusesMapping = createSelector(
```

Comment:
why you removed this selector?

Reply:
because the select id for `downloadStatusAdapter` entity is now message's id - that's the point of this PR